sources/p987.6/sections/03-defs_and_terms.adoc

```diff
@@ -0,0 +1,67 @@
+
+== Terms and definitions
+
+=== Definitions
+
+For the purposes of this document, the following terms and definitions
```

Comment:
This boilerplate is automatically generated, please remove.

Reply:
Ok, fixed.